source/MaterialXRuntime/Private/PvtApi.h

```diff
@@ -394,6 +478,10 @@ class PvtApi
     FileSearchPath _searchPaths;
     FileSearchPath _implementationSearchPaths;
     FileSearchPath _textureSearchPaths;
+    FileSearchPath _materialXDefinitionPaths;
```

Comment:
Is this one use? Is is never set anywhere as far as I can see?

What is the difference between _materialXDefinitionPaths and _materialXCoreDefinitionPaths?

Reply:
According to Bernard:
materialXCoreDefinitionPaths are: stdlib, pbrlib etc. materialXDefintionPaths come from the Materialx env variable ("MATERIALX_SEARCH_PATH")

Reply:
So then what is the difference between `getMaterialXCoreDefinitionPaths()` and the existing `getSearchPath()`?

Reply:
It seems that the search path points to the application and library paths, not the specific library paths. @bernardkwok may know more.

Reply:
We may be able to get away with not have materialXDefinitionPaths() as the API exists in MaterialX core already. I'm not sure if we have this access in all places. We could leave it out for now if the current context includes materialx access (which is true for the export case).